src/DependencyInjection/DI/src/ServiceLookup/ILEmit/ILEmitResolverBuilder.cs

```diff
@@ -185,6 +190,7 @@ protected override object VisitScopeCache(ServiceCallSite scopedCallSite, ILEmit
         protected override object VisitConstant(ConstantCallSite constantCallSite, ILEmitResolverBuilderContext argument)
         {
             AddConstant(argument, constantCallSite.DefaultValue);
+
```

Comment:
(nit) I believe this can be removed.
```suggestion
```

Reply:
Must have been part of the cherry-pick 😃

Reply:
The blank line was introduced in https://github.com/dotnet/extensions/pull/2501, but then removed in https://github.com/dotnet/runtime/pull/42152 (which looks like wasn't cherry-picked since it is a different repo).